tests/LightResults.Extensions.Operations.Tests/LightResults.Extensions.Operations.Tests.csproj

```diff
@@ -1,34 +0,0 @@
-<Project Sdk="Microsoft.NET.Sdk">
-
-    <PropertyGroup>
-        <TargetFrameworks>net481;net6.0;net6.0;net8.0;net9.0</TargetFrameworks>
```

Comment:
I added this one so that the tests could run against .NET Framework, since the library targets .NET Standard 2.0. The existing Tests project only targets .NET 6 at the lowest

Reply:
Yea, unfortunately, in GitHub Actions which run on Linux, I found no way to target 4.81. It just fails.

https://github.com/actions/setup-dotnet also doesn't support installing Framework versions.